### ISSUE TITLE
Hyperparameter settings for fine-tuning on PromptKD

### ISSUE THREAD

**Issue by Ld-Witt** [xjjxmu/TextRefiner#2]
Thank you for your innovative work!
I found that the base-to-novel generalization results of **PromptKD** in your paper are somewhat different from the original PromptKD paper.
**I AM JUST CURIOUS** that whether you have modified the original parameter settings of PromptKD to adapt it to your work (e.g., sample size, encoders or random seed)?
This is **NOT** to question your work! I just want to know what kind of experimental settings lead to such a difference.

**Comment by xjjxmu** [comment]
To explain the difference, PromptKD is an unsupervised distillation method designed to enhance the generalization of prompt learning and assumes all samples are available but unlabeled. However, we focus on the generalization of prompt learning in few-shot scenarios. 

Therefore, we just utilized PromptKD as the current state-of-the-art framework for visual prompt learning based on distillation and validate that our method can consistently improve performance within this sota visual prompt learning paradigm.

And, the performance reported in our work corresponds to the case with a limited number of samples and seems somewhat different from the original PromptKD paper which use all samples. Apart from the difference in sample quantity due to the scenario, all other training strategy and hyperparameters all remain consistent.

Thank you for your attention! I hope this helps with your understanding.

**Comment by xjjxmu** [comment]
Yes, we utilized few-shot setup to reproduce the PromptKD baseline and PromptKD with TextRefiner.
